test/test_utils.py

```diff
@@ -44,41 +44,37 @@ def _check_checkpoint_sequential(
         model,
         module_lists_to_compare,
         num_chunks,
-        *inputs
+        input,
     ):
 
         # not checkpointed
-        if not isinstance(inputs, tuple):
-            inputs = (inputs,)
-        out = model(*inputs)
+        out = model(input)
         out_not_checkpointed = out.data.clone()
         model.zero_grad()
         out.sum().backward()
         grad_not_checkpointed = {
             name: param.grad.data.clone()
             for name, param in model.named_parameters()
         }
-        input_grad_not_checkpointed = [i.grad.data.clone() for i in inputs]
+        input_grad_not_checkpointed = input.grad.data.clone()
```

Comment:
Can you use `.detach()` here instead of `.data` please?

Reply:
Sure! I agree with you.